quinn-proto/src/connection/mod.rs

```diff
@@ -315,6 +315,8 @@ pub struct Connection {
     abandoned_paths: FxHashSet<PathId>,
 
     iroh_hp: Option<iroh_hp::State>,
+    #[cfg(test)]
+    panic_on_transport_error: bool,
```

Comment:
ufff, is there any chance we can avoid this?

Reply:
I'm open to ideas!
We *could* make `Connection::handle_event` return a `Result<(), ...>` instead of `()`. This would make it possible for the `TestEndpoint` to have this config instead, but would require everyone to handle the `Error` correctly, and there might also be refactoring necessary, since we probably don't want to accidentally introduce short-circuiting  where there wasn't any before.

I mainly added it this way, since I wanted to test this & this was the easiest way to do so.